plugins/lsp/default.nix

```
@@ -206,12 +207,13 @@ in
 
                 options = {
                   inherit (cfg.keymaps) silent;
+                  inherit desc;
                 } // actionProps;
               }
             );
         in
-        (mkMaps "vim.diagnostic." cfg.keymaps.diagnostic)
-        ++ (mkMaps "vim.lsp.buf." cfg.keymaps.lspBuf)
+        (mkMaps "vim.diagnostic." "Lsp diagnostic" cfg.keymaps.diagnostic)
+        ++ (mkMaps "vim.lsp.buf." "Lsp buf" cfg.keymaps.lspBuf)
```

If you're touching these lines, could you drop the unnecessary parens?
```suggestion
        mkMaps "vim.diagnostic." "Lsp diagnostic" cfg.keymaps.diagnostic
        ++ mkMaps "vim.lsp.buf." "Lsp buf" cfg.keymaps.lspBuf
```

---

Resolved in https://github.com/nix-community/nixvim/pull/2598